.github/workflows/spectesting.yml

```diff
@@ -0,0 +1,66 @@
+on:
+  push:
+    branches:
+      - 'spec-test-*'
+    paths-ignore:
+      - 'CHANGELOG.md'
+      - 'website/**'
+
+jobs:
+  setup-packet:
+    if: github.repository == 'hashicorp/vagrant-blackbox'
+    runs-on: self-hosted
+    name: Build Packet Instance
+    steps:
+      - name: Create packet instance
+        run: .ci/spec/create-packet.sh
+        working-directory: ${{github.workspace}}
+
+  setup-hosts:
+    if: github.repository == 'hashicorp/vagrant-blackbox'
+    runs-on: self-hosted
+    name: Vagrant-Spec Start Hosts
+    needs: setup-packet
+    strategy:
```

Comment:
No strategy should be used in this job

Reply:
This is what is used to set up the Vagrant host and guests on packet. The strategy here is passed along into environment variables so vagrant-spec can bring up the right host and guest for the downstream job to provision and run test with.